eng/pipelines/libraries/build-job.yml

```diff
@@ -104,6 +106,13 @@ jobs:
                 sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/testhost
                 targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/testhost
 
+            - ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
+              - task: CopyFiles@2
```

Comment:
I think we should stop thinking about this and start thinking about a single job for these targets. Build the product, tests and run tests as part of the same job. We already do that for some other frameworks like netfx by using `runTests: true` parameter.

Reply:
I don't think we should. Running the tests should not be part of same job as we want to run them on different OS versions

Reply:
They will be running on different OS versions because we send them to helix and helix runs them. We never run them as part of the build machine. Running them as part of the build job will avoid uploading/downloading and will make the build simpler and much reliable.